clickhouse_client_settings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" ?>
+<config>
+    <max_partitions_per_insert_block>10000</max_partitions_per_insert_block>
+    <!-- Use all memory we have for data loading queries -->
+    <max_memory_usage>0</max_memory_usage>
+    <!-- Use disk for sorting and grouping when query grows bigger then 40Gb -->
+    <max_bytes_before_external_sort>40G</max_bytes_before_external_sort>
```

Comment:
this may help for the future but I am not sure it is currently needed as there are not high cost aggregations on prod running but it will probably help v2g_scored aggregation which is not done at the moment.

Reply:
Please note that these settings apply for the client session (for data loading only), not the server.

Reply:
@mkarmona FYI the script template you've shared [here](https://github.com/opentargets/genetics/issues/337#issuecomment-496557904) has:
```xml
<max_bytes_before_external_sort>80000000000</max_bytes_before_external_sort>
<max_bytes_before_external_group_by>80000000000</max_bytes_before_external_group_by>
```

Reply:
Yep. I faced the same you are time ago but I didn't remember. too many things at the same time...

Reply:
setting `max_memory_usage` to `0` is not a good practice.

Reply:
Yes, I agree.

Reply:
> Yep. I faced the same you are time ago but I didn't remember. too many things at the same time...
Yes, I understand.

BTW it seems to be important to have this on. in my case (`max_memory_usage=0`), the setting prevented the server from crashing when the query result did not fit the memory

Reply:
Interesting then. I will include it in my config